### PR TITLE
Module name package namespace

### PR DESCRIPTION
**Pull request by notlee**
- Do not return package namespace when returning `getModuleName`.
- Rename `getModuleName` as `getComponentName`.

See commits for more details

### CODE REVIEW

File: lib/helpers/files.js [hunk 1]
```diff
@@ -70,11 +70,12 @@ function getMainJsPath(cwd) {
...
 }
 
-function getModuleName(cwd) {
+function getComponentName(cwd) {
 	return getPackageJson(cwd)
 		.then(pkgJson => {
 			if (pkgJson) {
-				return pkgJson.name;
+				const packageName = pkgJson.name;
+				return packageName ? packageName.split('/').pop() : packageName;
```
Comment (JakeChampion) [inline lib/helpers/files.js:78]: do we require components be namespaced?

Comment (notlee) [reply lib/helpers/files.js:78]: no, most are but some aren't, the tests cover other cases :)

Comment (notlee) [reply lib/helpers/files.js:78]: `'o-thing'.split('/').pop() === 'o-thing';`
`'acme/o-thing'.split('/').pop() === 'o-thing';`

Comment (notlee) [reply lib/helpers/files.js:78]: I guess this means we continue to require component names to be unique, even if they are published to different places

**Review by JakeChampion — APPROVED** [review]

**Comment by JakeChampion** [comment]
How come the package-lock.json file has changed? We havent' modified any dependencies in this pull-request 😕

**Comment by JakeChampion** [comment]
> How come the package-lock.json file has changed? We havent' modified any dependencies in this pull-request 😕

It was because a different version of npm had been used 👍

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v11.0.0-beta.5 :tada:

**Pull request merged**